### DISCUSSION TITLE
Is this project still being actively supported

### DISCUSSION THREAD

**Discussion by brettonw** [mKeRix/room-assistant#1526] [General]
I'm doing a tradeoff comparison between this and ESPresence. It'd be super cool if they worked together, but I can't get Room Assistant to work at all, and don't see any new conversations about it.

**Comment by tyjthomas** [comment]
I was wondering the same thing - looks like there hasn't been an update in a while after pretty frequent updates in the past. Any news if this is still being developed or has this project died?

**Comment by husnainali09** [comment]
The developer has abandoned this project, I believe he is now working for espresense team.

**Comment by tyjthomas** [comment]
Thanks for letting me know. On May 14, 2024, at 7:45 AM, 5898Kinghuzi ***@***.***> wrote:﻿
The developer has abandoned this project, I believe he is now working for espresense team.
